### ISSUE TITLE
EXEC_BAD_ACCESS when trying to delete message using custom menu actions 

### ISSUE THREAD

**Issue by Sufi-Al-Hussaini** [jessesquires/JSQMessagesViewController#1177]
I'm getting `EXEC_BAD_ACCESS` when trying to delete a message using custom menu actions.
After enabling `Zombie Objects`, this is the error I see:
>JSQMessages[27654:181624] *** -[JSQMessagesCollectionViewCellIncoming release]: message sent to deallocated instance 0x7ff9cb107d50

I was able to reproduce this with the demo app. 
###### Steps to reproduce: 
* Add this to `[DemoMessagesViewController customAction]`:
```
- (void)customAction:(id)sender
{
    NSLog(@"Custom action received! Sender: %@", sender);

        JSQMessagesCollectionViewCell *cell = sender;
        NSIndexPath *indexPath = [self.collectionView indexPathForCell:cell];
        [self.demoData.messages removeObjectAtIndex:indexPath.item];
        [self.collectionView deleteItemsAtIndexPaths:@[indexPath]];
}
```
* Run the demo app. Push via storyboard -> delete any message -> back.

Is there something wrong with my code or is this a bug?

**Comment by Sufi-Al-Hussaini** [comment]
Related to #1026 .
Apologies for not having gone through earlier issues thoroughly before opening this one.
I'm using the 'develop' branch now, waiting for v7.1.2. :)

**Comment by jessesquires** [comment]
:+1: will be fixed in 7.1.2
